modules/ROOT/pages/mule-server-notification.adoc

```diff
@@ -0,0 +1,284 @@
+= Mule Server Notifications
+
+ifndef::env-site,env-github[]
+
+include::_attributes.adoc[]
+
+endif::[]
+
+:keywords: server notifications, log, downtime, monitor
+
+Mule provides an internal notification mechanism that you can use to access changes that occur on the Mule Server, such as adding a flow component, a request for authorization failing, or Mule starting. You can set up your agents or flow components to react to these notifications.
+
+== Configuring Notifications
+
+Message notifications provide a snapshot of all information sent into and out of the Mule Server. Mule fire these notifications whenever  it receives or sends a message. These additional notifications have some impact on performance, so they are disabled by default. To enable message notifications, you set the type of messages you want to enable using the `<notifications>` element in your Mule configuration file. You also register the notification listeners and associate interfaces with specific notifications.
+
+For example, first you create beans for the notification listeners in your spring configuration file, specifying the class of the type of notification you want to receive:
```

Comment:
I would write in "a spring configuration file (for example, notification-beans.xml)". I'm not sure anyway if this makes it clearer.